content/jobs/js2021.md

```diff
@@ -0,0 +1,23 @@
+---
+position: open
+banner: JavaScript Developer
+sub: Leading our front-end development process
```

Comment:
At 8 people, are we all "leading"? 🤔

Reply:
I made a revision suggestion to change it to `Collaborating on our front-end development process` here: https://github.com/oddbird/oddleventy/pull/134/files